test_scripts/WebEngine/WebEngineProjection/005_OnExitApplication_RESOURCE_CONSTRAINT_unregister.lua

```diff
@@ -13,16 +13,20 @@
 --  a. Web app is unregistered
 --  b. Mobile session is disconnected
 ---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
```

Comment:
@ydementieiev Please rewrite test script description in one style in all scripts of this PR. I would propose to use description style which is used in this script.

Reply:
@aderiabin [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/9271d45b0b4e176fad4cd4070eeb060facb11f43) look please